scapy/layers/http.py

```diff
@@ -298,6 +303,8 @@ def post_dissect(self, s):
             elif "compress" in encodings:
                 import lzw
                 s = lzw.decompress(s)
+            elif "br" in encodings:
+                s = brotli.decompress(s)
```

Comment:
This will crash Scapy when the `brotli` module is not available. I will suggest to use a `is_brotli_available`, set it to `True` on a successful import and use it in `elif "br" in encodings and is_brotli_available:`. Otherwise, this looks good to me.